Base/QTCore/qSlicerCoreApplication.cxx

```diff
@@ -902,7 +902,7 @@ void qSlicerCoreApplication::handleCommandLineArguments()
 
     // Set 'sys.executable' so that Slicer can be used as a "regular" python interpreter
     this->corePythonManager()->executeString(
-          QString("import sys; sys.executable = '%1'; del sys").arg(this->applicationFilePath()));
+          QString("import sys; sys.executable = '%1'; del sys").arg(QStandardPaths::findExecutable("PythonSlicer")));
```

Comment:
Let's do a conditional on the Qt versionm `QStandardPaths` is available only with Qt >= 5.0

Reply:
ok